platform/packages/finance/src/price/mod.rs

```diff
@@ -180,7 +180,7 @@ where
     }
 
     /// Add two prices rounding each of them to 1.10-18, simmilarly to
-    /// the precision provided by cosmwasm::Decimal.
+    /// the precision provided by cosmwasm_std::Decimal.
```

Comment:
It might be useful to make it a link, so that generated documentation can refer to it properly. Also useful for inline documentation in the IDE.
```suggestion
    /// the precision provided by CosmWasm's [`Decimal`](sdk::cosmwasm_std::Decimal).
```